spec/operators/takeWhile-spec.ts

```diff
@@ -364,4 +364,88 @@ describe('takeWhile', () => {
 
     expect(sideEffects).to.deep.equal([0, 1, 2]);
   });
+
+  it('should not emit errors sent from the source *after* it found the first value in reentrant scenarios', () => {
```

Comment:
if i revert the change in takeWhile, this test case isn't failing. to make it fail, i had to use:
```ts
        tap({
          next: (x) => {
            if (x === 'b') {
              subject.error(new Error('reentrant shennanigans'));
            }
          },
        })
```

Reply:
Seems this was fixed by a later commit (sorry, revisiting this)